index.html

```diff
@@ -0,0 +1,262 @@
+
+          <ul id="profs1">
+            <li id="cannopy1">CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+
+          <p>
+            A daily selection of privately personalized: no accounts or sign-ups
+            required
+          </p>
+          <ul class="button">
+            <li><button type="button">html</button></li>
+            <li><button type="button">css</button></li>
+            <li><button type="button">javascript</button></li>
+          </ul>
+
+          <div class="hidden1">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+
+        <div class="tonic">
+          <div><img src="images/snap1.png" alt="snap" /></div>
+          <div>
+            <h2 id="stories-header1">Tonic</h2>
+          </div>
+
+          <div>
+            <ul id="profs2">
+              <li id="cannopy2">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+
+          <div>
+            <p>
+              A daily selection of privately personalized: no accounts or
+              sign-ups required
+            </p>
+          </div>
+          <div>
+            <ul class="button">
+              <li><button type="button">html</button></li>
+              <li><button type="button">css</button></li>
+              <li><button type="button">javascript</button></li>
+            </ul>
+          </div>
+
+          <div class="hidden2"><button type="button">See Project</button></div>
+        </div>
+
+        <div class="tonic">
+          <img src="images/snap3.png" alt="snap" />
+          <h2 id="stories-header2">Multi-post stories</h2>
+
+          <ul id="profs3">
+            <li id="cannopy3">CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+
+          <p>
+            A daily selection of privately personalized: no accounts or sign-ups
+            required
+          </p>
+          <ul class="button">
+            <li><button type="button">html</button></li>
+            <li><button type="button">css</button></li>
+            <li><button type="button">javascript</button></li>
+          </ul>
+          <div class="hidden3">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </section>
+      <section class="abt">
+        <div class="about">
+          <div class="content-left">
+            <div>
+              <h2>About</h2>
+              <h2>Myself</h2>
+            </div>
+
+            <p>
+              Hello I'm a software developer! I can help you build your
+              project,feature or website.Look through some of my work and
+              experience! If you like what you see and have the project you need
+              coded,don't hesitate to contact me.
+            </p>
+
+            <a id="connect1" href="index.html">LET'S CONNECT</a>
```

Comment:
- [OPTIONAL] The other titles (e.g., "Let's connect")  should uses a header tag `hx (examples: <h2>,<h3>,<h4>, etc...)`. This way will be semantically correct and the screen readers and other technologies will thank you.